cmd/common/common.go

```diff
@@ -87,5 +87,6 @@ func NewClient(cfg *config.StorageConfig, logger *log.Logger) (storage.TargetSto
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
nit: unneeded?

Reply:
I _think_ this was just due to running `go fmt`. I actually prefer the separation for readability but happy to change if  ppl feel strongly.